app/components/leagueStandingTable/leagueStandingTable.tsx

```diff
@@ -0,0 +1,37 @@
+import { Table } from "../baseComponents";
+import { getWikipediaContestantDataFetcher } from "@/app/utils/wikiFetch";
+import generateListOfContestantRoundLists from '@/app/generators/contestantRoundListGenerator'
+import { getCompetingEntityList } from "@/app/utils/wikiQuery"
+
+import styles from "./leagueStandingTable.module.scss";
+
+export default async function LeagueStandingTable({ contestantsScores }:{contestantsScores: any}){
+    type rowState = any[];
+    const tableColNames: rowState = ["Rank", "Name", "Score"];
+    const tableRows: rowState = [];
+    const tableData = {
+        colsNames: tableColNames,
+        rows: tableRows
+    }
+
+    contestantsScores.forEach((contestantData:any) =>{
+        const { content: { props }} = contestantData;
```

Comment:
I think you will find that if you use `generateContestantRoundScores` you wouldn't need to do this

Reply:
Sorry, I couldn't stop thinking about this and I wanted to expand on this a bit. Looking a tad deeper, I was realizing what has been done in here to date and technically there are two "functions" called `generateContestantRoundScores` so that would certainly make anyone confused at this suggestion. Moreover they kind of do different things 😅 

_(as a slight aside, I would recommend anyone who finds this message to go take a look and try to tease apart what the difference between the two functions is. It's a bit subtler than you might think, and could serve as an interesting understanding of how code evolves over time)_

_Anyway_... I would probably suggest, depending on our need here, that we use one or both of them and perhaps refactor one or both to be a bit less confusing for the future.

Also ICYMI either one of the functions should result in something that looks a bit like this example
```
League Stnading : {
    "rounds": [
        {
            "round": 5,
            "contestantRoundData": [
                {
                    "name": "Antoinette",
                    "roundScore": 70,
                    "totalScore": 670
                },
                {
                    "name": "Duncan",
                    "roundScore": 80,
                    "totalScore": 660
                }
            ]
        },
        {
            "round": 0,
            "contestantRoundData": [
                {
                    "name": "Duncan",
                    "roundScore": 140,
                    "totalScore": 140
                },
                {
                    "name": "Antoinette",
                    "roundScore": 140,
                    "totalScore": 140
                }
            ]
        }
    ]
}
```